plugins/modules/s3_logging.py

```diff
@@ -102,16 +107,28 @@ def verify_acls(connection, module, target_bucket):
         "Permission": "FULL_CONTROL",
```

Comment:
While handling exceptions for the get_bucket_acl here, I think I'd be a little more relaxed with something like:

```
if not module.params.get("acl"):
    module.warn(f"Unable to fetch Bucket ACLs ({e})")
    return False
module.fail_json_aws(e, msg="Failed to fetch target bucket ACL")
```